wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/ProfileDeviceDetailsView.swift

```diff
@@ -142,7 +148,7 @@ struct ProfileDeviceDetailsView: View {
                     DeviceView(viewModel: viewModel).titleView
                 }
                 ToolbarItem(placement: .navigationBarTrailing) {
-                    SwiftUI.Button(
+                    Button(
```

Comment:
@caldrian This won't work since we have a class named Button. Needs SwiftUI. in front for it to work.

Reply:
I declared a `typealias` at the top of the file.